src/Promitor.Integrations.Sinks.Statsd/Configuration/StatsdSinkConfiguration.cs

```diff
@@ -5,5 +5,8 @@ public class StatsdSinkConfiguration
         public string Host { get; set; }
         public int Port { get; set; } = 8125;
         public string MetricPrefix { get; set; }
+        public string FormatterType { get; set; } = StatsdFormatterTypes.DEFAULT;
+        public string Namespace { get; set; }
+        public string Account { get; set; }
```

Comment:
```suggestion
        public GenevaConfiguration Geneva { get; set; }
```

Let's move this to an object so that your YAML is:
```yaml
geneva:
  namespace: abc
  acount: xyz
```